tests/unit/CheckoutFormTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    define('_PS_VERSION_', 'TEST_VERSION');
+}
+
+class CheckoutFormTest extends PHPUnit_Framework_TestCase
+{
+    private $list_of_expiration_year;
+
+    public function setup()
+    {
+        $checkout_form = new CheckoutForm();
+        $this->list_of_expiration_year = $checkout_form->getListOfExpirationYear();
+    }
+
+    public function testGetListOfExpirationYear_getListOfExpirationYear_totalNumberOfExpirationYearMustBe11()
+    {
+        $this->assertEquals(11, count($this->list_of_expiration_year));
+    }
+
+    public function testGetListOfExpirationYear_getListOfExpirationYear_theBeginningOfExpirationYearMustBeCurrentYear()
+    {
+        $current_year = date('Y');
+
+        $beginning_of_expiration_year = $this->list_of_expiration_year[0];
+
+        $this->assertEquals($current_year, $beginning_of_expiration_year);
+    }
+
+    public function testGetListOfExpirationYear_getListOfExpirationYear_theEndingOfExpirationYearMustBeNextTenYears()
```

Comment:
@nimid same as https://github.com/omise/omise-prestashop/pull/11/commits/4cf775f11525b93fabed57f75ff5b7ab697186f4#r95548688

`testGetListOfExpirationYear_retrieveTheLastElementOfAnArray_theResultMustBeNextTenYears` will describe for those variables and we can get rid of them.